src/components/Form/Widgets/SelectWidget.tsx

```diff
@@ -58,11 +73,38 @@ export const SelectWidget = ({
 				}}
 				disabled={disabled}
 				options={selectOptions}
-				isOptionEqualToValue={(option, value) => option.value === value.value}
-				getOptionLabel={(option) => option.label}
-				getOptionDisabled={(option) => option.disabled}
+				isOptionEqualToValue={(option, value) => isEqual(option, value)}
+				getOptionLabel={(option) =>
+					Array.isArray(option)
+						? option.map((o) => o.label).join(', ')
+						: option.label
+				}
+				getOptionDisabled={(option) =>
+					Array.isArray(option)
+						? option.every((o) => o.disabled)
+						: option.disabled
+				}
 				renderInput={(params) => <TextField {...params} />}
-				onChange={change}
+				onChange={(_event, selected) => {
+					if (Array.isArray(selected)) {
+						const val = selected
+							.map((item) => {
+								if (item && typeof item === 'object' && 'value' in item) {
+									return item.value;
```

Comment:
I guess because we're using RJSF we can guarantee that all items have a `value`?

Reply:
No this is Autocomplete typing, that's the default behaviour

```
NonNullable<{
    disabled: boolean;
    value: any;
    label: string;
    schema?: RJSFSchema | undefined;
} | {
    disabled: boolean;
    value: any;
    label: string;
    schema?: RJSFSchema | undefined;
}[]> | ({
    disabled: boolean;
    value: any;
    label: string;
    schema?: RJSFSchema | undefined;
} | {
    disabled: boolean;
    value: any;
    label: string;
    schema?: RJSFSchema | undefined;
}[])[]
```

Reply:
Huh, gotcha